metrotranscom/doorway
# feat: add various content updates

This PR addresses:

* #833
* #834
* #835


## Description

This updates content for the Get Started help page, the What to Expect page in common app, and Terms at the end of the common app flow.

**Technical note:** the Get Started page is oddly structured, with raw HTML markup containing individual strings. This makes it cumbersome to work with as well as difficult to translate with any embedded elements like links. I made a partial conversion to Markdown like our other content pages, but I recommend we file a followup issue to convert the rest of the page content to Markdown.

## How Can This Be Tested/Reviewed?

Visit those pages to ensure all the content has been updated in various languages.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
